Dockerfile

```diff
@@ -0,0 +1,16 @@
+# from node
+FROM node:latest
+
+
+RUN curl -f https://get.pnpm.io/v6.16.js | node - add --global pnpm
+RUN npm install typescript -g
```

Comment:
Can you remove this line? Typescript is installed as devDependencies with a specific version